elections.md

```diff
@@ -101,4 +91,16 @@ treated as “brand free” activities.
 ## Emeritus Term
 
 Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+the steering committee.
+
+## Steering committee and election officer recusal
+
+Steering committee members and the appointed election offers pledge to recuse
```

Comment:
I'm not sure where this paragraph came from, or what the intention is?  I think it would be fine for steering committee members to nominate or endorse candidates?  In fact arguably they are in a fairly good position to identify any shortcomings of the existing committee, and hence good candidates to help address those shortcomings.  What's the motivation for preventing that here?

Reply:
This paragraph came from the "bootstrap recusal" paragraph, as well as the recusal from 2017 voting guide.

This is the level of impartiality the bootstrap committee held itself to.  I get your point, but I also have concerns that while all votes are equal, some endorsements are more equal than others.  I would rather let the community have its say without our bias.

I would like to hear from other members of steering on this.

Reply:
I will add specific language that members running for re-election are able to campaign for themselves.

I get the desire to use influence for community benefit, I just have concerns that the sword cuts both ways, and no amount of policy can prevent that.  Ultimately we can't enforce what people do in private, so as long as the asks / supportive nudges are private, I'm fine with it.